examples/features/sub-modules/inter-broker-bridge/pom.xml

```diff
@@ -12,37 +12,24 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <parent>
+    <groupId>org.apache.activemq.examples.modules</groupId>
+    <artifactId>broker-modules</artifactId>
+    <version>2.4.0-SNAPSHOT</version>
+  </parent>
+
   <groupId>org.apache.artemis.examples</groupId>
-  <artifactId>artemis-jms-bridge-parent</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
+  <artifactId>inter-broker-bridge-parent</artifactId>
```

Comment:
Can we not change the name this is just for JMS, eg there is a way to do this with amqp

Reply:
What I think I'll do is move the doc into the actual bridge example so that the inter-broker-bridge-parent can be used for other examples (e.g. AMQP, etc.) in the future.